<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hauptfenster</class>
 <widget class="QMainWindow" name="Hauptfenster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>260</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Einsatzleitsystem - Klient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Einsatzzeit</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Lokal</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtZeitLokal">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string notr="true">99.99.9999 99:99:99 ZZZZ</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>UTC</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLineEdit" name="txtZeitUTC">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string notr="true">99.99.9999 99:99:99 ZZZZ</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="Stundenmodus">
               <property name="text">
                <string>24h</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Position</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QStackedWidget" name="PositionStapel">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="Koordinate"/>
           <widget class="QWidget" name="KeineAPI">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;Keine GPS API gefunden.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Funkkenner</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="txtFunkkenner">
           <property name="text">
            <string>Heros Berlin 99/99/99</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>102</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>364</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>923</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Funk">
    <property name="title">
     <string>&amp;Funk</string>
    </property>
    <addaction name="action_Kenner_setzen"/>
    <addaction name="action_Alphabet"/>
   </widget>
   <widget class="QMenu" name="menu_Hilfe">
    <property name="title">
     <string>&amp;Hilfe</string>
    </property>
    <addaction name="action_Ueber"/>
    <addaction name="action_Ueber_Erweiterungen"/>
   </widget>
   <widget class="QMenu" name="menu_Vordrucke">
    <property name="title">
     <string>&amp;Vordrucke</string>
    </property>
   </widget>
   <addaction name="menu_Funk"/>
   <addaction name="menu_Vordrucke"/>
   <addaction name="menu_Hilfe"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Kenner_setzen">
   <property name="text">
    <string>&amp;Kenner setzen</string>
   </property>
  </action>
  <action name="action_Alphabet">
   <property name="text">
    <string>&amp;Alphabet</string>
   </property>
  </action>
  <action name="action_Ueber">
   <property name="text">
    <string>&amp;Über</string>
   </property>
  </action>
  <action name="action_Ueber_Erweiterungen">
   <property name="text">
    <string>Über &amp;Erweiterungen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
